### ISSUE TITLE
EntityFieldQueryException: Unknown field: field_dinner_user_load_total...

### ISSUE THREAD

**Issue by skovstedconsulting** [larskoeie/bf_dinner#2]
Hej Lars,
Når jeg forsøger at åbne fanen "Fællesspisning" får jeg fejlen:

`EntityFieldQueryException: Unknown field: field_dinner_user_load_total in EntityFieldQuery->addFieldCondition() (line 841 of /home/lisbjerg/public_html/skovstedbogfoering.dk/drupal/includes/entity.inc).`

Er der nogen parametre der skal sættes op udenfor modulet?
Håber du har tid til at give mig et hint her.
Vh. Kresten

**Comment by larskoeie** [comment]
Ja, alle felter skal tilføjes uden for modulet. Du skal lave en nodetype, min hedder 'dinner', og den skal have nogen felter, og så skal user-entity have nogen felter. De felter er beskrevet her https://github.com/larskoeie/bf_dinner/blob/master/bf_dinner.module#L43
